daemon/src/monitor.rs

```diff
@@ -675,7 +651,7 @@ impl xtra::Actor for Actor {
                 }
             },
             |e| async move {
-                tracing::warn!("Failed to re-initialize monitoring: {e:#}");
+                tracing::warn!("Failed to re-broadcast transactions: {e:#}");
```

Comment:
🙃 I think the message reading `Failed to re-initialize monitoring: {e:#}` given we combine the two blocks. I'd argue that re-broadcasting transactions is part of re-initializing the monitoring (but not the other way round).

Reply:
I like your suggestion better than mine :)